docs/gatsbyjs.md

```diff
@@ -73,3 +73,19 @@ npm run develop
 
 This will run the Gatsby app with hot-reloading on `http://localhost:8000` and
 use `https://dev.telescope.cdot.systems` as your backend api.
+
+Please note `npm run develop` will detect 99% of any issues that may occur when working on the front-end. It is highly advised to run `npm run build` to verify there are no issues.
+
+```
+npm run build
+```
+
+If the build has been completed without any issues, it can now be served using:
+
+```
+npm run serve
+```
+
+The built front-end will be available on `http://localhost:9000` and use `https://dev.telescope.cdot.systems` as your backend api.
```

Comment:
9000?

Reply:
Does no one get the terminal prompt for this?

Reply:
I see what it is: [`serve` uses 9000](https://www.gatsbyjs.com/docs/gatsby-cli/#serve) and [`develop` uses `8000`](https://www.gatsbyjs.com/docs/gatsby-cli/#develop)

Personally, I don't think it's worth telling people about `serve` since we never use it--our static assets are always served by nginx, and I think adding more ports to the mix confuses things.